### PR TITLE
[Add profile] -> [Adeboye Folaranmi]

### PR DESCRIPTION
**Pull request by Boye95**
If you're adding your profile, the title of this PR should be: `[Add profile] -> [Your Name]`

### CODE REVIEW

**Review by Mannuel25 — CHANGES_REQUESTED** [review]
Hello @Boye95 , thanks for contributing. However, some changes have been made since your last edit, could you pull the changes made to avoid conflicts, then add your own profile.

In addition, do ensure you add your profile to the end of column 25. The final changes should look like this: 

```
<!-- Start of column-26 -->
<td align="center">
  <a href="https://github.com/Boye95">
    <img
      src="https://avatars.githubusercontent.com/u/74235313?v=4"
      width="100px"
    />
    <br />
    <sub>Adeboye Folaranmi</sub>
  </a>
</td>
<!-- End of column-26 -->
```
If you've any issue doing that, kindly let me know.

**Comment by Boye95** [comment]
How do I go about this ? I already tried git pull and my local files haven't been updated yet.

**Comment by Evavic44** [comment]
Hey, @Boye95 try these steps to update your fork. If you have any questions or face errors, do send me a message back. 

First, do a git fetch to get the latest changes.
```
git fetch upstream
```
Next, merge the changes with your new branch.
```
git merge upstream/main
```
If you have any conflicts after merging the changes, fix the conflict and push back to your name branch.

```
git push origin add-Boye
```

**Comment by Boye95** [comment]
I followed the steps you outlined above and it seems something's is still wrong.

**Comment by Evavic44** [comment]
> I followed the steps you outlined above and it seems something's is still wrong.

You didn't fix the conflicts that's the reason. Don't worry, I'll fix and merge it.

**Comment by Boye95** [comment]
Thanks. I'd love to learn how to do it too.

**Comment by Evavic44** [comment]
> Thanks. I'd love to learn how to do it too.

Okay, lemme allow you to do it. 
Git conflicts usually occur when multiple people are making different commits to the same line. 
For example:

**Comment by Evavic44** [comment]
Do you want me to fix it? or you want to give it a try? 😁 @Boye95

**Comment by Boye95** [comment]
Thanks for the explanation but it seems the conflict has been resolved ?

File: Contributors.md [hunk 1]
```diff
@@ -306,6 +306,7 @@
     <sub>Williams Samuel</sub>
   </a>
 </td>
+<<<<<<< HEAD
```
Comment (Evavic44) [inline Contributors.md:309]: Delete both merge branch line of text

```
<<<<<<< HEAD 
```

**Review by Evavic44 — CHANGES_REQUESTED** [review]
@Boye95 There are currently 27 profiles in the contributors list, kindly add yours

File: Contributors.md [hunk 2]
```diff
@@ -319,5 +320,30 @@
   </a>
 </td>
 <!-- End of column-25 -->
+<!-- Start of column-26 -->
```
[the remark this answers is not in the document]
Comment (Boye95) [reply Contributors.md:323]: I'm already number 25 on the list

**Comment by Evavic44** [comment]
This issue has been marked as stale because of no activity.

**Pull request closed without merging**